includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -35,7 +35,7 @@ abstract public static function get_instance( $options = array() );
 	 * @return string         Value of the option
 	 */
 	protected static function get( $option ) {
-		return self::$options[ $option ];
+		return self::$options[ $option ] ?? null;
```

Comment:
Maybe we can update the PHPDOC so now this function can return NULL

Reply:
Nice catch! Update here 4371430